code/modules/arcade/prize_datums.dm

```diff
@@ -0,0 +1,343 @@
+
+var/global/datum/prizes/global_prizes = new
+
+/datum/prizes
+	var/list/datum/prize_item/prizes = list()
```

Comment:
What he means is that typecasted lists don't *actually* have a type. Leaving lists untyped makes searching/grep much simpler.